include/nccl_ofi_tuner.h

```diff
@@ -0,0 +1,106 @@
+#ifndef NCCL_OFI_TUNER_H_
+#define NCCL_OFI_TUNER_H_
+
+#include <linux/limits.h>
+#include <float.h>
+#include "nccl-headers/nvidia/tuner.h"
+#include "nccl_ofi_param.h"
+
+/*
+ * The plugin interface lets us tune the number of channels as well, but that
+ * can come later (once a proto+algo combination is chosen, we can compute the
+ * cost with different channel count and optimize for it.
+ */
+OFI_NCCL_PARAM_INT(tuner_num_channels, "TUNER_NUM_CHANNELS", 8);
+
+/*
+ * Latency in µsecs. Note, this is currently different from the network plugin's param for
+ * net latency by design. When we merge with the platform_data values, we will
+ * need to do some additional testing on the base case where a tuner is not
+ * loaded to make sure the same defaykts make sense across both paths, and
+ * combine the parameters. This parameter is meant for internal testing only and
+ * is not meant to be documented for users.
+ */
+OFI_NCCL_PARAM_INT(tuner_net_latency, "TUNER_NET_LATENCY", 20);
+
+/*
+ * With EFA, we expect a ~2µsec cost in the device and ~1µsec cost to write that
+ * completion up to the host stack.
+ */
+OFI_NCCL_PARAM_INT(tuner_net_comp_overhead, "TUNER_NET_COMP_OVERHEAD", 3);
+
+/* EFA unidirectional network bandwidth */
+#define NCCL_OFI_TUNER_INTERNODE_BW	(12.5 * 1024 * 1024 * 1024 * 1e-6) /* per rail */
```

Comment:
grumble, grumble.  Let's at least make sure we have a follow-up issue to clean this up.